Assets/Scripts/Gamestate/PlayerManager.cs

```diff
@@ -151,8 +148,30 @@ public void SetLayer(int playerIndex)
         SetLayerOnSubtree(meshBase, playerLayer);
     }
 
-    private void SetGunOffset(Transform offset)
+    private void SetGun(Transform offset)
     {
+        if (identity.items.Count > 0)
```

Comment:
This check is redundant. A foreach loop will terminate immediately if the collection it iterates over is empty..

Reply:
Nice catch!